recipes/oneworld/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "oneworld" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 52660254dd1941cebca3f3834df27064fc0f583da87c2358f45ce0ac1d4b46d9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - matplotlib
+    - pandas
+    - seaborn
+    - jinja2
+    - cartopy
+
+test:
+  imports:
+    - oneworld
+  requires:
```

Comment:
The source distribution does not come with the test files, so you can remove the `source_files` section along with the `requires`, since you aren't actually running `pytest` and aren't able to with the given setup.